### DISCUSSION TITLE
There are two home buttons

### DISCUSSION THREAD

**Discussion by dlwoals256** [MrGreensWorkshop/MrGreen-JekyllTheme#89] [Problems] [answered]
**Describe the bug**
there's two button of home and I attempted to fix _site/index.html line 169.

<li style="pointer-events: none;"><a href="/" class="active-page hover-effect"><i class="fa-fw fa fa-home" aria-hidden="true"></i>Home</a></li><li style="pointer-events: none;"><a href="/" class="active-page hover-effect"><i class="fa-fw fa fa-home" aria-hidden="true"></i>Home</a></li><li ><a href="/tabs/blog/" class=" hover-effect"><i class="fa-fw fa fa-pencil-square-o" aria-hidden="true"></i>Blog</a></li><li ><a href="/tabs/archive.html" class=" hover-effect"><i class="fa-fw fa fa-archive" aria-hidden="true"></i>Archive</a></li><li ><a href="/tabs/projects.html" class=" hover-effect"><i class="fa-fw fa fa-cog" aria-hidden="true"></i>Projects</a></li><li ><a href="/tabs/links.html" class=" hover-effect"><i class="fa-fw fa fa-link" aria-hidden="true"></i>Links</a></li><li ><a href="/tabs/about.html" class=" hover-effect"><i class="fa-fw fa fa-user-o" aria-hidden="true"></i>About</a></li></ul>

to

<li style="pointer-events: none;"><a href="/" class="active-page hover-effect"><i class="fa-fw fa fa-home" aria-hidden="true"></i>Home</a></li><li ><a href="/tabs/blog/" class=" hover-effect"><i class="fa-fw fa fa-pencil-square-o" aria-hidden="true"></i>Blog</a></li><li ><a href="/tabs/archive.html" class=" hover-effect"><i class="fa-fw fa fa-archive" aria-hidden="true"></i>Archive</a></li><li ><a href="/tabs/projects.html" class=" hover-effect"><i class="fa-fw fa fa-cog" aria-hidden="true"></i>Projects</a></li><li ><a href="/tabs/links.html" class=" hover-effect"><i class="fa-fw fa fa-link" aria-hidden="true"></i>Links</a></li><li ><a href="/tabs/about.html" class=" hover-effect"><i class="fa-fw fa fa-user-o" aria-hidden="true"></i>About</a></li></ul>

after I do, I attempted on cmd 'bundle install' and 'bundle exec jekyll serve' to run to see it fixed.

but it looks perfectly same before I fixed the index.html
and I look up to the _site/index.html line 169, it doesn't changed...
can you fix this? plz help me X(

**Screenshots**

<img width="748" alt="Screen Shot 2022-06-15 at 19 57 41" src="https://github.com/devnovy/devnovy/blob/94f0883b57906943e07c28aa9efa8de7510a2aba/%ED%99%94%EB%A9%B4%20%EC%BA%A1%EC%B2%98%202022-06-20%20074850.png?raw=true">

**Expected behavior**

**To Reproduce**

**Desktop (please complete the following information):**

**Mobile devices (please complete the following information):**

**Additional context**

**Comment by MrGreensWorkshop** [comment, marked as answer]
I'm sorry I don't understand why you have 2 buttons. 
please watch the first video try to do same thing in the video, then please check if you have same problem?
 https://www.youtube.com/playlist?list=PLAymxPbYHgl-fFy5can7uZBMJtFWVcphD
 
I'm not sure if this helps,  _site/ folder is output folder, 
_site   ->  "This is where the generated site will be placed (by default) "
https://jekyllrb.com/docs/structure/

If you change something inside that folder and run jekyll serve, all changes will be reverted to previews state.

**Comment by dlwoals256** [comment]
I success to fix it by following your video! thx very much!!! have a good day :)
